components/collector/tests/source_collectors/gitlab/test_job_runs_within_time_period.py

```diff
@@ -91,24 +99,24 @@ async def test_jobs_not_deduplicated(self):
         response = await self.collect(get_request_json_return_value=self.gitlab_jobs_json)
         expected_entities = [
             {
+                "build_date": str(just_now.date()),
+                "build_datetime": just_now,
+                "branch": "main",
+                "build_result": "failed",
                 "key": "3",
                 "name": "job3",
-                "url": self._job3_url,
-                "build_status": "failed",
-                "branch": "main",
                 "stage": "stage",
-                "build_date": str(just_now.date()),
-                "build_datetime": just_now,
+                "url": self._job3_url,
             },
             {
-                "key": "4",
-                "name": "job3",
-                "url": self._job3_url,
-                "build_status": "failed",
                 "branch": "main",
-                "stage": "stage",
                 "build_date": str(yesterday.date()),
                 "build_datetime": just_now - timedelta(days=1),
+                "build_result": "failed",
+                "key": "4",
```

Comment:
hmm, interesting that key 4 is job 3 in this test case only

Reply:
Copy-paste error apparently. Fixed.